src/spatio_temporal_voxel_grid.cpp

```diff
@@ -260,24 +265,30 @@ void SpatioTemporalVoxelGrid::operator()(const \
     float mark_range_2 = obs._obstacle_range_in_m * obs._obstacle_range_in_m;
     const double cur_time = ros::WallTime::now().toSec();
 
-    pcl::PointCloud<pcl::PointXYZ>::const_iterator it;
-    for (it = obs._cloud->points.begin(); it < obs._cloud->points.end(); ++it)
+    const sensor_msgs::PointCloud2& cloud = *(obs._cloud);
+    sensor_msgs::PointCloud2ConstIterator<float> iter_x(cloud, "x");
+    sensor_msgs::PointCloud2ConstIterator<float> iter_y(cloud, "y");
+    sensor_msgs::PointCloud2ConstIterator<float> iter_z(cloud, "z");
+
+    for (; iter_x !=iter_x.end(); ++iter_x, ++iter_y, ++iter_z)
```

Comment:
Please add `iter_x, iter_y, iter_z` just for completion

Reply:
Added